acquia/cli
# CLI-863: Fix Drush alias download

**Motivation**

Fixes #1191

**Proposed changes**


- Match string instead of integer
- Fix tests to cover this case

**Testing steps**


1. Follow the [contribution guide](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#building-and-testing) to set up your development environment or [download a pre-built acli.phar](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#automatic-dev-builds) for this PR.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Ensure `remote:aliases:download` works.
